app/helpers/application_helper.rb

```diff
@@ -206,7 +206,7 @@ def searchable?
 
   def auto_complete_search(name, val, options = {})
     path = options[:path] || send("#{controller_name}_path")
```

Comment:
@domcleal that's intentional.

The auto_complete_search should not be appended here because, by doing so, we bypass the [override logic](https://github.com/theforeman/foreman/blob/develop/app/helpers/filters_helper.rb#L14).  Also, as far as I can tell, it's not needed but please correct me if I'm wrong.